README.md

```diff
@@ -103,8 +103,8 @@ Please note for this to work you should create/update `~/.gradle/gradle.properti
     signing.password=
     signing.secretKeyRingFile=
 
-### Publishing the jars without signing to a local repository ###
```

Comment:
Originally @ewencp added this for helping other developers work around signing. I will leave it to him for reviewing this.